onebusaway-watchdog-webapp/src/main/resources/org/onebusaway/watchdog/application-context-webapp.xml

```diff
@@ -92,7 +92,7 @@
     </bean>
 	
 	<bean id="configurationServiceClient" class="org.onebusaway.util.impl.configuration.ConfigurationServiceClientFileImpl">
-		<constructor-arg type="java.lang.String" value="/var/lib/oba/config.json"/>
+		<constructor-arg type="java.lang.String" value="/opt/nyserda/tds/config.json"/>
```

Comment:
Please review your local changes before committing, to prevent issues like this.  The default should stay /var/lib/oba/config.json